cmd/kubeadm/app/cmd/reset.go

```diff
@@ -147,11 +147,8 @@ func (r *Reset) Run(out io.Writer) error {
 		glog.Errorf("[reset] failed to unmount mounted directories in /var/lib/kubelet: %s\n", string(umountOutputBytes))
 	}
 
-	fmt.Println("[reset] removing kubernetes-managed containers")
-	dockerCheck := preflight.ServiceCheck{Service: "docker", CheckIfActive: true}
-	execer := utilsexec.New()
-
-	reset(execer, dockerCheck, r.criSocketPath)
+	glog.V(1).Info("[reset] removing kubernetes-managed containers")
```

Comment:
Above we hard-code `"/var/run/dockershim.sock"`, can you change so it's using the constant from the API?

Reply:
This is addressed in this PR I believe: https://github.com/kubernetes/kubernetes/pull/64911